docs/source/java/flight_sql_jdbc_driver.rst

```diff
@@ -114,6 +114,21 @@ parameters are:
      - null
      - When TLS is enabled, the password for the certificate store
 
+   * - tlsRootCerts
+     - null
+     - Path to PEM-encoded root certificates for TLS - use this as
+       an alternative to ``trustStore``
+
+   * - clientCertificate
```

Comment:
It's a bit odd that these new parameters are raw PEM files instead of JKS files like the other TLS-related parameters.

Reply:
Hi @jduo - I was trying to add a convenience factor.  For me, it is easier to work with PEM files - I added tlsRootCerts for folks that prefer PEM.  I had been using PEM files with ADBC already - and it seemed so convenient, I wanted the option for JDBC...